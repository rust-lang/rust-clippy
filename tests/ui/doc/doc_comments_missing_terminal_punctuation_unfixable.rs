#![feature(custom_inner_attributes)]
#![rustfmt::skip]
#![warn(clippy::doc_comments_missing_terminal_punctuation)]
// Only line doc comments are provided with suggestions.
//@no-rustfix

/**
//~^ doc_comments_missing_terminal_punctuation
 * Block doc comments work
 *
 */
struct BlockDocComment;
