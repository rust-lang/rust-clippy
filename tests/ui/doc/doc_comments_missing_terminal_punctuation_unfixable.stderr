error: doc comments should end with a terminal punctuation mark
  --> tests/ui/doc/doc_comments_missing_terminal_punctuation_unfixable.rs:7:1
   |
LL | / /**
LL | |
LL | |  * Block doc comments work
LL | |  *
LL | |  */
   | |___^
   |
   = note: `-D clippy::doc-comments-missing-terminal-punctuation` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::doc_comments_missing_terminal_punctuation)]`

error: aborting due to 1 previous error

